### DISCUSSION TITLE
Enhancing Navigation on Profile Pages with a Back Button Feature

### DISCUSSION THREAD

**Discussion by DamacekraLD** [ppy/osu#31270] [Ideas] [answered]
I’m not sure if this is important or not, but here’s my thought:
When I visit my profile page on osu! client, I usually want to check beatmaps like "most played" or "favorite beatmaps." After clicking on the details of a beatmap, I notice that there is no option to go back to my profile. Instead, I have to close the beatmap overlay and click the profile menu again to return. Adding a "previous" or "back" or usually i use the "esc" hotkeys button similar to a web browser would make navigation more efficient.

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue: https://github.com/ppy/osu/issues/5063
